File: hw_06/Java/java_oopwork06/src/geometry/Rectangle.java
public class Rectangle {

    private double width;
    private double height;

    public Rectangle(double width, double height) {
        this.width = width;
        this.height = height;
    }

    public Rectangle() {
        this(1.0, 1.0);
    }

    public double getWidth() {
        return width;
    }

    public void setWidth(double width) {
        this.width = width;
    }

    public double getHeight() {
        return height;
    }

    public void setHeight(double height) {
        this.height = height;
    }

    public double area() {
        return width * height;
    }

    public void showRecInfo() {
        System.out.println("\nRectangle:");
        System.out.println("Width = " + width);
        System.out.println("Height = " + height);
        System.out.println("Area = " + area());
    }
}

File: hw_06/Python/hw_06/geometry/Rectangle.py
class Rectangle:
    def __init__(self, width:float=1.0, height:float=1.0):
        self.__width = width
        self.__height = height

    def get_width(self):
        return self.__width
    
    def set_width(self, width:float):
        self.__width = width

    def get_heigth(self):
        return self.__height
    
    def set_heigth(self, height:float):
        self.__height = height

    def area(self):
        return self.__width * self.__height
    
    def info(self):
        return f"Rectangle: Width {self.__width:.2f}, Height: {self.__height:.2f}, Area {self.area():.2f}"